src/main/resources/application.properties

```diff
@@ -21,8 +21,11 @@ spring.jpa.hibernate.ddl-auto=update
 #spring.jpa.hibernate.ddl-auto=none
 spring.jpa.properties.hibernate.event.merge.entity_copy_observer=allow
 
+
+spring.config.import=optional:file:${HOME}/geoweaver/application.properties
+
```

Comment:
Caused by: java.sql.SQLSyntaxErrorException: Table 'geoweaver.HOST' doesn't exist
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:121)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeInternal(ClientPreparedStatement.java:912)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeQuery(ClientPreparedStatement.java:968)
	at com.zaxxer.hikari.pool.ProxyPreparedStatement.executeQuery(ProxyPreparedStatement.java:52)
	at com.zaxxer.hikari.pool.HikariProxyPreparedStatement.executeQuery(HikariProxyPreparedStatement.java)
	
### Does not show publicuser host:
<img width="483" alt="image" src="https://github.com/ESIPFed/Geoweaver/assets/45752727/a6c617a0-f6db-4ddf-b47a-389118ddbdc7">

### Adding a host fails
Caused by: java.sql.SQLSyntaxErrorException: Table 'geoweaver.HOST' doesn't exist
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:121)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeInternal(ClientPreparedStatement.java:912)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeQuery(ClientPreparedStatement.java:968)
	at com.zaxxer.hikari.pool.ProxyPreparedStatement.executeQuery(ProxyPreparedStatement.java:52)
	at com.zaxxer.hikari.pool.HikariProxyPreparedStatement.executeQuery(HikariProxyPreparedStatement.java)


### Environment table is empty
<img width="1292" alt="image" src="https://github.com/ESIPFed/Geoweaver/assets/45752727/e32ee1c9-16c4-476c-924c-2fbb6b5396a9">